### PR TITLE
Support Laravel 8.

### PR DESCRIPTION
**Pull request by CraicOverflow89**
Updated dependencies and GitHub tests.

### CODE REVIEW

**Review by willpower232 — CHANGES_REQUESTED** [review]
please can you remove the travis file and badge and add trailing newlines to the workflow file and the Dockerfile

**Review by willpower232 — APPROVED** [review]

File: src/Chrome.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function __construct($binary = 'chrome-pdf', $processClass = null)
      */
     public function createProcess($cmdline)
     {
-        return new $this->{'processClass'}($cmdline);
+        return new $this->{'processClass'}([$cmdline]);
```
Comment (Josh-G) [inline src/Chrome.php]: there's a docblock issue here, `createProcess` is doc'd as receiving a string, but actually [receives an array](https://github.com/SynergiTech/chrome-pdf-php/blob/2dc7b9f7a914f8c9d6307eba5bbc54f738e6c2bf/src/Chrome.php#L212)
as such, we don't need to wrap it

**Review by Josh-G — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Setup PHP
-      uses: shivammathur/setup-php@v1
-      with:
-        php-version: ${{ matrix.php-versions }}
-        coverage: xdebug
+        laravel: [5, 6, 7, 8]
```
Comment (Josh-G) [inline .github/workflows/main.yml]: please drop references to `laravel`, as this library has no hard dependency on it
might be worth replacing with a version matrix on PHP and `symfony/process`

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+WORKDIR /package
+
+COPY composer.json ./
+
+RUN composer install
+
+COPY . .
+
+RUN vendor/bin/phpunit --configuration phpunit.xml
```
Comment (Josh-G) [inline Dockerfile]: please can you change this to `RUN composer test`, which should run `phpunit` + `phpstan`

File: README.md [hunk 4]
```diff
@@ -1,6 +1,5 @@
 # PHP ChromePDF Renderer
 ![Unit tests](https://github.com/SynergiTech/chrome-pdf-php/workflows/Unit%20tests/badge.svg)
-[![Build Status](https://travis-ci.org/SynergiTech/chrome-pdf-php.svg?branch=master)](https://travis-ci.org/SynergiTech/chrome-pdf-php)
```
Comment (Josh-G) [inline README.md:3]: new status badge:
[Tests](https://github.com/SynergiTech/chrome-pdf-php/actions/workflows/main.yml)
```
[![Tests](https://github.com/SynergiTech/chrome-pdf-php/actions/workflows/main.yml/badge.svg)](https://github.com/SynergiTech/chrome-pdf-php/actions/workflows/main.yml)
```

File: phpstan.neon [hunk 5]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    paths:
+        - app
+    level: 5
+    ignoreErrors:
+        - '#Result of function abort \(void\) is used\.#'
+        - '#Method App\\Exceptions\\Handler::render\(\) should return Illuminate\\Http\\Response but returns Symfony\\Component\\HttpFoundation\\Response\.#'
```
Comment (Josh-G) [inline phpstan.neon]: Doesn't look like these are real errors in this library? Can you remove these?

File: phpstan.neon [hunk 6]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    paths:
+        - app
+    level: 5
```
Comment (Josh-G) [inline phpstan.neon]: Can you bump it up to 8?

File: phpstan.neon [hunk 7]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    paths:
+        - app
```
Comment (Josh-G) [inline phpstan.neon]: this should be `src`?

**Review by Josh-G — CHANGES_REQUESTED** [review]

**Comment by CraicOverflow89** [comment]
Is this okay @Josh-G?

**Review by Josh-G — APPROVED** [review]

**Pull request merged**